modules/animationqt/src/animationeditordockwidgetqt.cpp

```diff
@@ -360,6 +419,8 @@ AnimationEditorDockWidgetQt::AnimationEditorDockWidgetQt(
 
     toolBar->addSeparator();
 
+    animationView_->setFocus();
```

Comment:
Why?

Reply:
It was needed to get the key events to go to the animation controller. Might be better ways of doing this, setTabOrder? I removed it for now. I think that the dock widgets do not get focus when they are shown right now? The focus is still on the network editor when I tested it. 
Is probably another PR

Reply:
Fixed focus issue in latest commits